src/libs/Permissions.ts

```diff
@@ -17,11 +17,8 @@ function canUseDefaultRooms(betas: Beta[]): boolean {
     return betas?.includes(CONST.BETAS.DEFAULT_ROOMS) || canUseAllBetas(betas);
 }
 
-/**
- * IOU Send feature is temporarily disabled.
- */
-function canUseIOUSend(): boolean {
-    return false;
+function canUseIOUSend(betas: Beta[]): boolean {
+    return betas?.includes(CONST.BETAS.IOU_SEND) || canUseAllBetas(betas);
```

Comment:
Are we sure we want to leave this behind a beta? At Money2020 how will we make sure that the attendees are all on the `IOU_SEND` beta?

Reply:
Also, how will we make sure that the users at Money2020 are on the `BETAS_EXPENSIFY_WALLET` beta for the `canUseWallet` method just below this?

Reply:
After the discussions here https://expensify.slack.com/archives/C05RECHFBEW/p1696530879965409 we have agreed to remove the wallet and send money beta.